ecosystem/sep-0038.md

```diff
@@ -586,16 +588,18 @@ available at this endpoint past the `expires_at` expiration for the quote.
+| `id`                   | string                                                                        | The `id` specified in the request.                                                                                                                                                                      |
+| `expires_at`           | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | The date and time by which the anchor must receive funds from the client.                                                                                                                               |
+| `price`                | string                                                                        | The price offered by the anchor for one unit of `buy_asset` in terms of `sell_asset`. In traditional finance, `buy_asset` would be referred to as the base asset and `sell_asset` as the counter asset. |
+| `sell_asset`           | string                                                                        | The asset the client would like to sell. Ex. `USDC:G...`, `iso4217:ARS`                                                                                                                                 |
+| `sell_amount`          | string                                                                        | The amount of `sell_asset` to be exchanged for `buy_asset`.                                                                                                                                             |
+| `sell_delivery_method` | string                                                                        | (optional) The method by which the user plans to deliver an off-chain asset to the anchor. This will only be present if specified in the request.                                                       |
+| `buy_asset`            | string                                                                        | The asset the client would like to exchange for `sell_asset`.                                                                                                                                           |
+| `buy_amount`           | string                                                                        | The amount of `buy_asset` to be exchanged for `sell_asset`. `price * buy_amount = sell_amount` must be true up to the number of decimals required for `buy_asset`.                                      |
+| `buy_delivery_method`  | string                                                                        | (optional) The method by which the user plans to receive an off-chain asset from the anchor. This will only be present if specified in the request.                                                     |
```

Comment:
```suggestion
| `buy_delivery_method`  | string                                                                        | (optional) The method by which the user plans to receive an off-chain asset from the anchor. This will only be present if the parameter was specified in the associated `POST /quote` request.                                                     |
```